libPhoneNumber-Demo/libPhoneNumber-Demo/GeocodingTableView.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Created by Rastaar Haghi on 7/17/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import SwiftUI
+import libPhoneNumber_iOS
+import libPhoneNumberGeocoding
+
+struct GeocodingTableView: View {
+    // Keep track of runtime statistics
+    var maxRuntime: CGFloat = 0.00
+    var minRuntime: CGFloat = 0.00
+    var totalRuntime: CGFloat = 0.00
+    var averageRuntime: CGFloat = 0.00
+    
+    init() {
+        for _ in 1...50 {
+            makeGeocodingAPICalls()
+        }
+        
+        self.maxRuntime = runtimeArray.max() ?? 0.0
+        self.minRuntime = runtimeArray.min() ?? 0.0
+        self.totalRuntime = 0.00
+        for i in 0..<500 {
+            self.totalRuntime += runtimeArray[i]
+            runtimeArray[i] = runtimeArray[i]/maxRuntime
+        }
+        self.averageRuntime = totalRuntime / CGFloat(runtimeArray.count)
+    }
+    var body: some View {
+        VStack {
+            Form {
+                Section(header: Text("This table makes 500 Geocoding API calls per page refresh.")) {
+                    List {
+                        ForEach(regionDescriptions, id: \.self) { pair in
+                            HStack {
+                                Text(pair[0]!)
+                                Spacer()
+                                Text(pair[1]!)
+                            }
+                        }
+                    }
+                }
+            }
+            Form {
+                Section(header: Text("Runtime Performance for Geocoding API Calls")) {
+                    LineGraph(dataPoints: runtimeArray)
+                        .stroke(Color.green, lineWidth: 2)
+                        .aspectRatio(16/9, contentMode: .fit)
+                        .border(Color.gray, width: 1)
+                        .padding()
+                }
+                Section(header: Text("Statistics for Runtime Performance (in milliseconds)")) {
+                    List {
+                        Text("Average API Call Runtime: \(round(averageRuntime).description)")
+                        Text("Longest API Call Runtime: \(round(maxRuntime).description)")
+                        Text("Shortest API Call Runtime: \(round(minRuntime).description)")
+                        Text("Total Runtime: \(round(totalRuntime).description)")
+                    }
+                }
+            }
+        }
+        .navigationBarTitle("Large Set of Geocoding Calls")
+    }
+}
+
+struct GeocodingTableView_Previews: PreviewProvider {
+    static var previews: some View {
+        GeocodingTableView()
+    }
+}
+
+extension GeocodingTableView {
+    // Fetch Geocoding info for each number in phoneNumbers
+    func makeGeocodingAPICalls() {
+        var parsedPhoneNumber: NBPhoneNumber
+        for phoneNumber in phoneNumbers {
+            do {
+                let startTimer = DispatchTime.now()
+                parsedPhoneNumber = try phoneUtil.parse(phoneNumber, defaultRegion: "US")
```

Comment:
Just make this a let. You don't use it outside of this scope

Reply:
done.